### ISSUE TITLE
(suggestion):math.random()  : precision for doc, + seed

### ISSUE THREAD

**Issue by FabriceNeyret** [josdejong/mathjs#1430]
is your random based on the JS random, or do you implement your own algorithm ?
in the later case:
-  It means that it behaves the same way on all browsers, which is a big plus, that should be mentioned in the doc. 
- Then it would be useful to mention which RNG is used.
- Then you could implement a really missing feature: setting the seed (or did I missed it ?)

thanks !

**Comment by josdejong** [comment]
mathjs uses the [`seed-random`](https://www.npmjs.com/package/seed-random) library under the hood. You can configure the seed using 
```js
math.config({ randomSeed: ... })
```

**Comment by FabriceNeyret** [comment]
cool ! any idea which RNG this lib is using ?

-> in the doc, section random, I suggest to add these crucial info that a lot are seeking for: 
* random algo is identical on all browsers/OS, not like the JS native one
* can be seeded (cf math.config)
* the RNG name, if you have

thanks !

**Comment by josdejong** [comment]
Makes sense to improve the docs. Are you able to create a pull request improving the docs @FabriceNeyret?
